dataflow/snippets/tests/test_batch_write_storage.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import sys
+import uuid
+
+from google.cloud import storage
+
+import pytest
+
+from ..batch_write_storage import write_to_cloud_storage
+
+
+bucket_name = f'test-bucket-{uuid.uuid4()}'
+storage_client = storage.Client()
+
+
+@pytest.fixture(scope="function")
+def setup_and_teardown():
+    try:
+        bucket = storage_client.create_bucket(bucket_name)
+        yield
+    finally:
+        bucket.delete(force=True)
+
+
+def test_write_to_cloud_storage(setup_and_teardown):
+    sys.argv = ['', f'--output=gs://{bucket_name}/output/out-']
+    write_to_cloud_storage()
+
+    blobs = list(storage_client.list_blobs(bucket_name))
```

Comment:
As per `write_to_cloud_storage`, output format is `.txt` files. Can you add a test-case to ensure that `.txt` is generated ?

Reply:
oh good idea, thanks. Added.

Reply:
I put it in the same test, should it be a separate test?

Reply:
you can use the same test case.